src/shared/project-peer-review-item/ProjectPeerReviewItem.tsx

```diff
@@ -0,0 +1,91 @@
+import { i18n } from '@lingui/core';
+import { Box, Divider, Grid, makeStyles, Theme, Typography } from '@material-ui/core';
+import { CSSProperties } from '@material-ui/core/styles/withStyles';
+import graphql from 'babel-plugin-relay/macro';
+import React, { useCallback } from 'react';
+import { useFragment } from 'react-relay/hooks';
+import { ProjectPeerReviewForm } from 'src/shared/project-peer-review-form';
+import { ProjectCommentFormData } from 'src/shared/project-peer-review-form/types';
+import { ProjectPeerReviewOutput } from 'src/shared/project-peer-review-output';
+import { FCProps } from 'src/shared/types/FCProps';
+import { Styles } from 'src/shared/types/Styles';
+import { useSaveProjectComment } from './saveProjectComment.mutation';
+import { ProjectPeerReviewItem_projectReview$key } from './__generated__/ProjectPeerReviewItem_projectReview.graphql';
+interface OwnProps {
+  projectReview: ProjectPeerReviewItem_projectReview$key;
+}
+
+type Props = FCProps<OwnProps> & StyleProps;
+
+const fragment = graphql`
+  fragment ProjectPeerReviewItem_projectReview on ProjectReviewNode {
+    id
+    reviewee {
+      firstName
+    }
+    project {
+      name
+    }
+    ...ProjectPeerReviewOutput_projectReview
+    comment {
+      ...ProjectPeerReviewForm_projectComment
+    }
+  }
+`;
+
+export function ProjectPeerReviewItem(props: Props) {
+  const projectReview = useFragment(fragment, props.projectReview);
+  const classes = useStyles(props);
+  const saveProjectComment = useSaveProjectComment();
+  const projectReviewId = projectReview.id;
+  const onSubmit = useCallback(
+    (input: ProjectCommentFormData | null) => {
```

Comment:
null??

Reply:
Yup, input can be null as well
That's because we can accept null in form as comment (as projectReview.comment is nullable) thus the data on submit callback can be null as well